packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -158,9 +158,9 @@
         <div class="consent--section" v-show="displayConsent">
           <p class="consent--section-text">
             By clicking “SIGN UP” I agree to the applicable Free Trial terms in
-            <a href="/terms">Okta’s Terms of Service</a> during my use of the
+            <a href="/terms" target="_blank">Okta’s Terms of Service</a> during my use of the
             Free Trial Service and Okta’s
-            <a href="https://www.okta.com/privacy-policy">Privacy Policy</a>. I further agree that
+            <a href="https://www.okta.com/privacy-policy" target="_blank">Privacy Policy</a>. I further agree that
```

Comment:
Should these also have `rel="noopener noreferrer"`?

Reply:
I'll create a ticket to go over all of our new links and validate that this is the case, I know we have a couple of these missing.

Reply:
https://oktainc.atlassian.net/browse/OKTA-360351?atlOrigin=eyJpIjoiOWNhZTBiMDI1MWNiNDQ1OTgyZTE0YzgwNTYxNzBjOGEiLCJwIjoiaiJ9